docs/testing.md

```diff
@@ -0,0 +1,91 @@
+# Testing considerations
```

Comment:
if you want have an own testing for documentation, can you please add a section for developer tests about
- who-to enable annotations to ensure to run on a certain DB or howto disable this.